tests/heif/heif_read.c

```diff
@@ -0,0 +1,23 @@
+/**
+ * Simple test case that confirms the failure of using `gdImageCreateFromHeif`
+ * with a NULL pointer.
+ */
+
+
+#include "gd.h"
+#include "gdtest.h"
+
+int main()
+{
+	gdImagePtr im;
+    FILE *fp;
```

Comment:
looks like the file has mixed indentation.  you want tabs everywhere.

Reply:
Yeah, seems that both me and VSCode didn't caught that.